### DISCUSSION TITLE
Rejected for Student developer pack

### DISCUSSION THREAD

**Discussion by AbdullahDaGoat** [community/community#57991] [GitHub Education]
### Hi everyone,

I have tried and gotten rejected 8 times for github student developer pack, i used my school issue ID from clarkson secondary in mississauga ontario and it does not work. It says, that it was unsuccesful and got rejected, and that i tried 8 times even though the last time i tried besides today was April 2023. Please help. if you need any info i can give, idk what to do.

**Comment by jge162** [comment]
Are you issued a school email? If so update your account to use that first. Second upload a copy of your school transcripts as your school id clearly does not show years on it it sounds like. Good luck.

**Comment by AbdullahDaGoat** [comment]
Ill try the school email thing but my school Id does show clearly the date, my homerrom, my photo, my school, my student number, and some additional info.

**Reply by jge162** [reply]
With all that said, school ID is not working. So you have to try something else.

**Reply by jge162** [reply]
I never used school ID to get personally approved and have recommended to many people to use transcripts and they have all been approved.

**Comment by mhz1990** [comment]
I'm having the same problem, I must have tried 10 times now. I attend school online and keep receiving an error stating that my transcript and enrollment forms don't say it's online even though they very clearly do

**Comment by Walter-Hartwell-White** [comment]
> ### Hi everyone,
> I have tried and gotten rejected 8 times for github student developer pack, i used my school issue ID from clarkson secondary in mississauga ontario and it does not work. It says, that it was unsuccesful and got rejected, and that i tried 8 times even though the last time i tried besides today was April 2023. Please help. if you need any info i can give, idk what to do.
> 
> ### A suggestion...(optional)
> _No response_



> ### Hi everyone,
> I have tried and gotten rejected 8 times for github student developer pack, i used my school issue ID from clarkson secondary in mississauga ontario and it does not work. It says, that it was unsuccesful and got rejected, and that i tried 8 times even though the last time i tried besides today was April 2023. Please help. if you need any info i can give, idk what to do.
> 
> ### A suggestion...(optional)
> _No response_

I have also rejected for student developer pack 9 times and now GitHub is not accepting my dated id card also. 
Please reply @[GitHubEducation](https://github.com/)

**Comment by samus-aran** [comment]
Hi @AbdullahDaGoat 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

**Reply by ajaynegi45** [reply]
Dear @samus-aran ,

I hope this message finds you well. My name is Ajay Negi, and I am writing to seek your assistance regarding my application for the Github Student Developer Pack. I have applied for the pack seven times now, but unfortunately, my application has been rejected each time due to the unavailability of an academic email address.

As a passionate student developer, I am eager to access the valuable resources and tools offered by the Github Student Developer Pack. I strongly believe that these resources will play a crucial role in enhancing my academic and career pursuits. However, my college does not provide academic email addresses, which has been a significant hurdle in obtaining approval for the pack.

In my previous attempts, I have uploaded my college ID card and my 1st Semester Exam admit cards as a means of verification. Regrettably, these documents were not accepted. I understand the importance of verification for the program, and I assure you that I am genuinely a student seeking to utilize these resources for educational purposes.

To verify my student status, I only have my Student ID card and 1st and 2nd-semester exam Admit cards, which contain my name and other official details. Additionally, I have a username and password for the Student Portal access, which my university provides to its students.

I genuinely appreciate your dedication to supporting students in their development journey, and I believe that your assistance will make a significant difference in my academic and professional growth.

I kindly request your understanding and assistance in finding a suitable solution to this predicament. I am open to any alternative methods of verification or documentation that you may suggest, as I am determined to comply with the program's guidelines and requirements.

I would be extremely grateful if you could reconsider my application for the Github Student Developer Pack or offer any alternative solution that would allow me to access these valuable resources. Your help and understanding in this matter would mean a great deal to me.

Thank you for taking the time to read this message, and I eagerly await your response. Please feel free to reach out to me by simply reply it. If you require any further information or documentation.

Thank you once again for your time and consideration.

Sincerely,

Ajay Negi

**Reply by acastkat** [reply]
Hello, i got instantly rejected too, I don´t have a school email and github keeps asking me for it. Where can I send my provatory documentation to get the pack? Thanks
